packages/tools/src/tools/annotation/EllipticalROITool.ts

```diff
@@ -868,6 +874,27 @@ class EllipticalROITool extends AnnotationTool {
         }
       );
 
+      // draw center point, if "centerPointRadius" configuration is valid.
+      const minRadius = Math.min(
+        Math.abs(canvasCorners[0][0] - canvasCorners[1][0]) / 2, // horizontal radius
+        Math.abs(canvasCorners[0][1] - canvasCorners[1][1]) / 2 // vertical radius
+      );
```

Comment:
Can we put this calculations in the if statement below so that we don't add extra stuff when the radisu is 0?